src/app/modules/teacher/teacher-home-page/teacher-home-page.component.html

```diff
@@ -0,0 +1,14 @@
+<div mat-card style="margin: 3% 10% 1.5% 10%">
+  <br>
+  <button mat-flat-button color="primary" style="width: 100%;margin: 1% auto 1.5% auto" (click)="navigate('subscribe')">
```

Comment:
You can look at the navbar to see how routing is done in Angular without needing to define a function https://github.com/Clean-CaDET/platform-tutor-ui-web/blob/master/src/app/modules/navbar/navbar.component.html#L2